### ISSUE TITLE
Cannot Assign any Hotkeys...

### ISSUE THREAD

**Issue by AKTanara** [Belphemur/SoundSwitch#679]
**Describe the bug**
Cannot Assign any Hotkeys...

**Version**
SoundSwitch v5.10.0.19681
Windows 10 Build 19043.1055

**To Reproduce**
1. Go to 'Settings' -> 'Playback' or 'Recording' or 'Profiles' -> '+Add'
2. Click on 'HotkeyBox' to change the hotkey
3. no matter what combination is pressed, content of the box gets red and doesn't accept anything.

**Screenshots**
![2021-06-21_021553](https://user-images.githubusercontent.com/20730424/122689368-5968b280-d237-11eb-8f19-27d6b41fe2f8.jpg)
![2021-06-21_021802](https://user-images.githubusercontent.com/20730424/122689375-6a192880-d237-11eb-9358-5feb6071d412.jpg)

**Additional context**
- Default hotkey combination works but no change is possible.
- I can add hotkeys in other applications and can assign shortcut keys to Shortcuts.
- In Windows Sandbox (Virtual Isolated Environment in Windows 10) I can add hotkey without any problems.

**Comment by AKTanara** [comment]
Found The Problem...
It war a rare combination of conflicting softwares but I do document for people who might have same issue:
I have used [SharpKeys](https://github.com/randyrants/sharpkeys) UI to re assign some of my keyboard keys in windows registry.
![2021-06-21_035116](https://user-images.githubusercontent.com/20730424/122691416-c59de300-d244-11eb-8000-5e57fbd0d570.jpg)

I Used SharpKeys again to remove those re-assignments and now I could change those Hotkeys in SoundSwitch.
After making those changes in SoundSwitch, I re-assigned my keyboard keys in SharpKey again and now I have both desired combination of Hotkeys in SoundSwitch and key re-assignments on my keyboard...
![2021-06-21_035733](https://user-images.githubusercontent.com/20730424/122691426-d5b5c280-d244-11eb-94a4-b6ed0792f935.jpg)

I deeply appreciate the effort made for development of this software.
It is a huge help for me.
I leave in a country that doesn't have access to the international payment systems. If it wasn't the case I would love to donate.
Regards
